nav2_semantic_map_server/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>nav2_semantic_map_server</name>
+  <version>0.0.1</version>
+  <description>
+    Dummy version of semantic map server
+  </description>
+  <maintainer email="alexey.merzlyakov@samsung.com">Alexey Merzlyakov</maintainer>
+  <license>Apache-2.0</license>
+  <license>BSD-3-Clause</license>
```

Comment:
Not sure why BSD is here, there's not enough of anything here to have an additional license

Reply:
This was added by mistake. In latest commit there is no `package.xml` for `costmap_filter_info_pub` (formerly `nav2_semantic_map_server`)